oxy/poc_translator.py

```diff
@@ -0,0 +1,73 @@
+from ruamel.yaml import YAML
+from functools import reduce
+
+
+yaml: YAML = YAML(typ="safe")
+
+'''
+pub fn decode_accumulator_status(data: &[u8]) -> HashMap<u8, f32> {
+    let mut result = HashMap::new();
+    result.insert(1, (pd::big_endian(&data[0..2], 8) as f32) / 10.0);
+    result.insert(
+        2,
+        pd::twos_comp(pd::big_endian(&data[2..4], 8) as u32, 16) as f32 / 10.0,
+    );
+    result.insert(3, pd::big_endian(&data[4..6], 8) as f32);
+    result.insert(4, data[6] as f32);
+    result.insert(5, data[7] as f32);
+    result
+}
+'''
```

Comment:
Right off the bat this whole thing should be encapsulated in a class. I think there is a much better way of structuring this program around objects that would make it all easier to read and understand